recipe/activate-gcc.sh

```diff
@@ -158,11 +158,11 @@ _tc_activation \
   "CONDA_TOOLCHAIN_HOST,@CHOST@" \
   "CONDA_TOOLCHAIN_BUILD,@CBUILD@" \
   cc cpp gcc gcc-ar gcc-nm gcc-ranlib \
-  "CPPFLAGS,${CPPFLAGS:-${CPPFLAGS_USED}}" \
-  "CFLAGS,${CFLAGS:-${CFLAGS_USED}}" \
-  "LDFLAGS,${LDFLAGS:-${LDFLAGS_USED}}" \
-  "DEBUG_CPPFLAGS,${DEBUG_CPPFLAGS:-${DEBUG_CPPFLAGS_USED}}" \
-  "DEBUG_CFLAGS,${DEBUG_CFLAGS:-${DEBUG_CFLAGS_USED}}" \
+  "CPPFLAGS,${CPPFLAGS_USED} ${CPPFLAGS:-}" \
```

Comment:
Change to "CPPFLAGS,${CPPFLAGS_USED}"
and then add the following above line 156
```
[ -z "$CPPFLAGS" ] || CPPFLAGS_USED="${CPPFLAGS} ${CPPFLAGS_USED}"
```

Reply:
Thanks! I'll have to do all of them I think.

Reply:
You can also do,
```
CPPFLAGS,${CPPFLAGS_USED}${CPPFLAGS:+ }${CPPFLAGS:-}
```

Reply:
see https://github.com/conda-forge/ctng-compiler-activation-feedstock/pull/68